### ISSUE TITLE
[BotBug] using accented letter give weird result

### ISSUE THREAD

**Issue by bobonov** [theramenrobotdiscocode/lemur-engine#12]
Describe the bug
When user input has accented chars, these chars gets deleted.
This generally seem to happen to every char that is converted to an entity.

To Reproduce
A simple template like the following
You said: <star/>

User Says: 'I say èff fìfo àge'
Bot Responds With Error: 'You said: ff f fo ge '

Expected behavior
I expect to see the exact user input 'èff fìfo àge'

Something more:
I made a custom tag to display a html button and in this case the result is even weird.
The accented letter is displayed, but a space is added before, so "perché" became "perch è".
This happen only to the first entity.
When I use the tag inside a `<li></li>` the buttton tag get divided at first entity
Here follow a screenshot of same template outside and inside `<li>`
![immagine](https://user-images.githubusercontent.com/1412083/124800687-3b4cc300-df56-11eb-8388-0a723b944c76.png)

Template code is as follow
```
<set name="interesse">acquisto</set>
        Prima di Acquistare devi vendere?
        <btn value="prima vendere si valutazione">Sì, vorrei una valùtazione gratuita</btn>
        <btn value="prima vendere si autonomia">Sì, ma lo farò in autoàomia</btn>
        <btn value="prima vendere no">No</btn>
<condition name="interesse">
    <li value="acquisto">
        Prima di Acquistare devi vendere?
        <btn value="prima vendere si valutazione">Sì, vorrei una valùtazione gratuita</btn>
        <btn value="prima vendere si autonomia">Sì, ma lo farò in autoàomia</btn>
        <btn value="prima vendere no">No</btn>
    </li>
</condition>
```

I add also the custom tag.
[BtnTag.php.txt](https://github.com/theramenrobotdiscocode/lemur-engine/files/6778983/BtnTag.php.txt)

**Comment by elizaibeth** [comment]
thanks will investigate asap - but the system has not been tested for foreign chars so i can see this is prob a bug

**Comment by elizaibeth** [comment]
I have written a test AIML category...
`<?xml version="1.0"?>
<aiml>
		<category>
			<pattern>SAY FOREIGN *</pattern>
			<template><star/></template>
		</category>
</aiml>
`
I have confirmed...
`User: say foreign Sì, vorrei una valùtazione gratuita
Bot: S vorrei una val tazione gratuita`

In the database...
`*************************** 2. row ***************************
                id: 2
   conversation_id: 1
       category_id: NULL
client_category_id: NULL
    parent_turn_id: NULL
              slug: say-foreign-si-vorrei-una-valutazione-gratuita
             input: say foreign Sì, vorrei una valùtazione gratuita
            output: S vorrei una val tazione gratuita
            status: O
            source: human
        created_at: 2021-08-01 11:44:29
        updated_at: 2021-08-01 11:44:30
        deleted_at: NULL
`
It is saving it fine the the DB i believe it is cleaning it before it is output ...
Hopefully will be able to solve soon

**Comment by elizaibeth** [comment]
This has now been fixed and a unit test written to confirm the fix

`
User: say foreign Sì, vorrei una valùtazione gratuita
Bot: Sì vorrei una valùtazione gratuita
`

The fix is currently in develop branch and will be live in the next release

**Comment by elizaibeth** [comment]
Fix now in the release v1.3.0
